CHANGES.md

```diff
@@ -2,24 +2,19 @@
 
 ## edge-20.6.3
 
-This edge release is our release candidate for 2.8.1. It includes improvements
-to the multicluster support functionality as well as other fixes.
-
-* Controller
-  * Added an option to change the set of networks for which the proxy
-    should do discovery by IP address
-  * Upgraded the version of Grafana to 7.0.3
-  * Improved multicluster support to be able to work with gateways that
-    expose a hostname rather than a concrete IP address
-* Web UI
-  * Fixed a bug causing the Linkerd checks to fail when ran from the
-    dashboard (thanks @alex-berger!)
-* Internal
-  * Added an integration test for upgrading from edge to the current build
-  * Fixed a problem with the `install-pr` script, causing it to not be
-    able to find expected docker images
-  * Introduced CI steps for producing a Chocolatey package
```

Comment:
What is the outcome of this? Are we producing chocalatey packages for releases? That we have a CI step isn't particularly meaningful to me...

Reply:
A Chocolatey package is produced each time a stable release is tagged.